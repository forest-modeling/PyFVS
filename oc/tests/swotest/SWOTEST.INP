SWO TEST               2  25129.0125.0FF  0  0  12  13   0319                      0.   0.   0.    0.
 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00
 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00
 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00 1.00
     00000 0.0 0.0 0.0 0.0 0.0   0.0   0.0   0.0   0.0   0.0
 15 2 0  5.0 40.00.80  1.0000 0.00  1
 17 2 0  5.0 40.00.80  1.0000 0.00  1
 81 5 0  5.0 25.00.80  1.0000 0.00  1
117 4 0  5.0 30.00.72  1.0000 0.00  1
122 3 0  5.0 30.00.64  1.0000 0.00  1
231 8 0  5.0 20.00.58  1.0000 0.00  1
242 7 0  5.0 30.00.90  1.0000 0.00  1
263 6 0  5.0 35.00.62  1.0000 0.00  1
31213 0  5.0 45.00.59  1.0000 0.00  1
35116 0  5.0 55.00.52  1.0000 0.00  1
361 9 0  5.0 40.00.61  1.0000 0.00  1
43110 0  5.0 25.00.55  1.0000 0.00  1
49217 0  5.0 30.00.75  1.0000 0.00  1
63111 0  5.0 35.00.53  1.0000 0.00  1
80512 0  5.0 25.00.62  1.0000 0.00  1
81514 0  5.0 30.00.47  1.0000 0.00  1
81815 0  5.0 35.00.59  1.0000 0.00  1
92018 0  5.0 30.00.75  1.0000 0.00  1
202 1 0  5.0 40.00.77 10.0000 0.00  2
202 1 0 10.0 65.00.74 10.0000 0.00  2
202 1 0 15.0 90.00.71 10.0000 0.00  2
202 1 0 20.0110.00.69 10.0000 0.00  2
202 1 0 25.0130.00.68 10.0000 0.00  2
202 1 0 30.0140.00.67 10.0000 0.00  2
202 1 0 35.0155.00.67 10.0000 0.00  2
