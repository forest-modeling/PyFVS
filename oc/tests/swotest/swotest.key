!!Suppose 2.02 oc 0 0
!!PP ppif.1viewOut
!!Top
Comment
Starting year for simulation is 2010
Ending year for simulation is   2060
Min and Max inventory years are 2010 2010
Common cycle length is          5
End
!!End
!!Stand SWOTEST 0
StdIdent
SWOTEST  Stand SWOTEST at test or
!!SK oc
Screen
StandCN
1
InvYear         2010
!!End
!!TK
NumCycle          10
!!End
!!C From Database~0 -1 -1 @ @ All 0
!!SW FreeForm
!!P .
!!K
Database
DSNIn
swotest.mdb
StandSQL
SELECT * 
FROM FVS_StandInit 
WHERE Stand_CN = '%Stand_CN%' 
EndSQL
TreeSQL
SELECT * 
FROM FVS_TreeInit 
WHERE Stand_CN = '%Stand_CN%' 
EndSQL
END
!!End
!!C From Database~1 -1 -1 @ @ All 0
!!SW FreeForm
!!P .
!!K
NOCALIB 
NOTRIPLE
!!End
Organon
!!C ORGANON in FVS: ORGInfo~2 2 -1 oc organon All 0
!!SW GPKeyword
!!P ORGANON in FVS: ORGInfo~keyword.organon.ORGInfo~0~0~1~1~.
!!K
ORGInfo            1         0         1         1
!!End
!!C ORGANON in FVS: ORGVols~3 2 -1 oc organon All 0
!!SW GPKeyword
!!P # 2 ORGANON in FVS: ORGVols~keyword.organon.ORGVols~8.  0.0000 Fmax 8.00 N.
!! oB~8.  0.0000 Fmax 8.00 NoB~32.  0.0000 Fmax 32.00 NoB~0.5  0.0000 Fma.
!! x 0.50 NoB~6.  0.0000 Fmax 6.00 NoB~.
!!K
ORGVols           8.        8.       32.       0.5        6.
!!End
End
!!C Build Standard Treelist and Cutlist~4 2 -1 oc base All 0
!!SW GPKeyword
!!P # 1 Build Standard Treelist and Cutlist~base_TreeAndCutLists~1~0.  0.0000 .
!! Fmax 0.00 NoB~0~0~0.  0.0000 Fmax 0.00 NoB~0~1~.
!!K
Treelist          0.                   0
!!Cutlist           0.                   0

*The following text sends the Treelist and Cutlist Tables to a database.
!Database
!!TREELIST           0         3        1          0         0         0         1
!Treelist          0.                   0
!Cutlist            1
!End
!!End
!!C Compute Stand Variables in Editor~6 2 -1 oc base All 0
!!SW BuildExpression
!!P Compute Stand Variables in Editor~UDComputes~.
!!K
Compute            0
BA=BBA
TPA=BTPA
QMD=BaDBH
TOPHT=BTopHt
TCUFT=BTCuFt
BDFT=BBdFt
End

 
 

!!End
DataBase
!!C Specify Output Database~5 2 -1 oc dbs All 0
!!SW GPKeyword
!!P Specify Output Database~dbs_database~FVSOut.xls~.
!!K
DSNOut
FVSOut.xls
!!End
!!C Build Compute Table in Database~7 2 -1 oc dbs All 0
!!SW GPKeyword
!!P Build Compute Table in Database~keyword.dbs.Compute~0~0~.
!!K
Compute            0         0
!!End
End
SPLabel
   All, &
   SOUTH_RIVER_2, &
   10002
Process
!!EndStand

STOP
!!G SOUTH_RIVER_2 1
!!S SWOTEST 0
!!G 10002 2
!!S SWOTEST 0
!!Subset
!!G All 0
!!End
