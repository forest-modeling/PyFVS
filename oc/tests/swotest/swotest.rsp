testswo.key 
testswo.fvs 
testswo.out 
testswo.trl 
testswo.sum 
testswo.chp 
