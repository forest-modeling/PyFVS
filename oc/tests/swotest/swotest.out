        FOREST VEGETATION SIMULATOR     VERSION      -- ORGANON BETA RELEASE                   RV:11/14/12    05-09-2013  15:32:27

----------------------------------------------------------------------------------------------------------------------------------

                                                OPTIONS SELECTED BY INPUT

KEYWORD FILE NAME: SWOTEST.KEY
----------------------------------------------------------------------------------------------------------------------------------
KEYWORD    PARAMETERS:
--------   -----------------------------------------------------------------------------------------------------------------------

COMMENT 
           Starting year for simulation is 2010
           Ending year for simulation is   2060
           Min and Max inventory years are 2010 2010
           Common cycle length is          5

END 

STDIDENT
           STAND ID= SWOTEST                            Stand SWOTEST at test or                                               

SCREEN     SUMMARY TABLE WILL BE PRINTED TO DATA SET REFERENCE NUMBER     6 AS RUN PROGRESSES.

STANDCN    DATA BASE CONTROL NUMBER=1

INVYEAR    INVENTORY YEAR= 2010

NUMCYCLE   NUMBER OF CYCLES= 10

DATABASE   DATABASE KEYWORDS:

DSNIN      DSN FOR INPUT CONNECTION IS swotest.mdb
           CONNECTION DATA BASE TYPE:ACCESS

STANDSQL   STANDSQL COMMAND FOR INPUT CONNECTION: swotest.mdb
           SELECT *
           FROM FVS_StandInit
           WHERE Stand_CN = '%Stand_CN%'

           STAND-LEVEL DATA BASE READ:
           INV_YEAR:               2010
           REGION:                    7
           FOREST:                   10
            COMPOSITE LOC:          710

********   FVS03 WARNING:  FOREST CODE INDICATES THE GEOGRAPHIC LOCATION IS OUTSIDE THE RANGE OF THE MODEL.  DEFAULT CODE IS USED.
           FOREST CODE USED IN THIS PROJECTION IS  610
           AGE:                      40
           ASPECT:                200.0
           SLOPE:                 45.00
           BASAL_AREA_FACTOR:       0.0
           INV_PLOT_SIZE:            1.
           BRK_DBH:               999.0
           NUM_PLOTS:                 2
           NONSTK_PLOTS:              0
           SAM_WT:             8.000000
           DG_MEASURE:                5
           SITE_SPECIES:           PSME MAPPED TO INTERNAL CODE: DF
           SITE_INDEX:            129.0 FOR SPECIES: DF
           MAX_SDI:               530.0
           MODEL_TYPE:                1
           END OF DATA BASE READ.

TREESQL    TREESQL COMMAND FOR INPUT CONNECTION: swotest.mdb
           SELECT *
           FROM FVS_TreeInit
           WHERE Stand_CN = '%Stand_CN%'

           COLUMN PROCESSING RESULTS:
           TREE_CN         WAS IGNORED
           STAND_CN        WAS IGNORED
           STAND_ID        WAS IGNORED
           TREE_ID         WAS BOUND
           PLOT_ID         WAS BOUND
           TREE_COUNT      WAS BOUND
           HISTORY         WAS BOUND
           SPECIES         WAS BOUND
           DIAMETER        WAS BOUND
           DIAMETER_HT     WAS IGNORED
           HT              WAS BOUND
           HTG             WAS BOUND
           HTTOPK          WAS BOUND
           CRRATIO         WAS BOUND
           DAMAGE1         WAS BOUND
           DAMAGE2         WAS BOUND
           DAMAGE3         WAS BOUND
           SEVERITY1       WAS BOUND
           SEVERITY2       WAS BOUND
           SEVERITY3       WAS BOUND
           TREEVALUE       WAS BOUND
           PRESCRIPTION    WAS BOUND
           AGE             WAS BOUND
           TI_PK           WAS IGNORED


           NOTE: INPUT SPECIES CODE (017     )WAS SET TO (WF  ) FOR THIS PROJECTION.
           NUMBER ROWS PROCESSED:   25

END        END OF DATA BASE OPTIONS.

NOCALIB    CALIBRATION USING GROWTH DATA FROM TREE RECORDS HAS BEEN SUPPRESSED FOR ALL SPECIES

NOTRIPLE

ORGANON    ALL CYCLES;  PERIOD LENGTH=  5

ORGANON    ORGANON KEYWORDS:

ORGINFO    ORGANON VARIANT (1=SWO,2=NWO,3=SMC)= 1; IS EVEN-AGED (0=NO,1=YES)= 1;
           USE MAX-SDI (0=NO,1=YES)= 1; USE CALIBS (0=NO,1=YES)= 1

ORGVOLS    ORGANON LOG TRIM IS  8.0 INCHES; MINIMUM LOG LENGTH IS  8.0 FEET; TARGET LOG LENGTH IS 32.0 FEET;
           BF STUMP HEIGHT IS  0.5 FEET; BF MIN TOP DIB IS  6.0 INCHES
           CF STUMP HEIGHT IS  0.0 FEET; CF MIN TOP DIB IS  0.0 INCHES

END        ORGANON EXTENSION ACTIVE, END OF OPTIONS.

TREELIST   DATE/CYCLE=    0; DATA SET REFERENCE NUMBER =  3.; HEADING SUPPRESSION CODE = 0.
           (0=WITH HEADING, OTHER VALUES=SUPPRESS HEADING).


            
           *The following text sends the Treelist and Cutlist Tables to a database.

DATABASE   DATABASE KEYWORDS:

TREELIST   TREE INFORMATION FROM OUTPUT SENT TO CONNECTION FVSOut.mdb
           SPECIES CODE OUTPUT FORMAT: NORMAL  
           CREATE TEXT FILE REPORT: YES

CUTLIST    CUTS INFORMATION FROM OUTPUT SENT TO CONNECTION FVSOut.mdb
           SPECIES CODE OUTPUT FORMAT: NORMAL  
           CREATE TEXT FILE REPORT: YES

END        END OF DATA BASE OPTIONS.

COMPUTE    DATE/CYCLE=    0; DEFINE THE FOLLOWING:

           BA=BBA
           TPA=BTPA
           QMD=BaDBH
           TOPHT=BTopHt
           TCUFT=BTCuFt
           BDFT=BBdFt

END


            
            
            
            

DATABASE   DATABASE KEYWORDS:

DSNOUT     DSN FOR OUTPUT CONNECTION IS FVSOut.xls
           CONNECTION DATA BASE TYPE:EXCEL

COMPUTE    COMPUTE VARIABLES SENT TO CONNECTION FVSOut.xls
           ADD NEW VARIABLES TO EXISTING TABLE: YES
           INCLUDE VARIABLES STARTING WITH UNDERSCORE: NO 

END        END OF DATA BASE OPTIONS.

SPLABEL    STAND POLICY LABEL SET:
           All, SOUTH_RIVER_2, 10002

PROCESS    PROCESS THE STAND.

********   FVS09 WARNING:  PLOT COUNTS DO NOT MATCH DATA ON THE DESIGN RECORD; DESIGN RECORD DATA USED.
                           PLOT COUNT=   1; NONSTOCKABLE COUNT=   0
----------------------------------------------------------------------------------------------------------------------------------

                                                OPTIONS SELECTED BY DEFAULT

----------------------------------------------------------------------------------------------------------------------------------

TREEFMT    (I4,T1,I7,F6.0,I1,A3,F4.1,F3.1,2F3.0,F4.1,I1,3(I2,I2),2I1,I2,2I3,2I1,F3.0)      
                                                                                           

DESIGN     BASAL AREA FACTOR=    0.0; INVERSE OF FIXED PLOT AREA=    1.0; BREAK DBH=  999.0
           NUMBER OF PLOTS=     2; NON-STOCKABLE PLOTS=     0; STAND SAMPLING WEIGHT=     8.00000
           PROPORTION OF STAND CONSIDERED STOCKABLE=  1.000

********   FVS14 WARNING:  HABITAT/PLANT ASSOCIATION/ECOREGION CODE WAS NOT RECOGNIZED; HABITAT/PLANT ASSOCIATION/ECOREGION SET TO DEFAULT CODE.

           PLANT COMMUNITY CODE USED IN THIS PROJECTION IS CWC221  

STDINFO    FOREST-LOCATION CODE=     610; HABITAT TYPE=452; AGE=   40; ASPECT AZIMUTH IN DEGREES= 200.; SLOPE=  45.%
           ELEVATION(100'S FEET)= 35.0;  REFERENCE CODE=     

SPECIES      PC      IC      RC      WF      RF      SH      DF      WH      MH      WB
SDI MAX     530.    530.    530.    530.    530.    530.    530.    530.    530.    530.

SPECIES      KP      LP      CP      LM      JP      SP      WP      PP      MP      GP
SDI MAX     530.    530.    530.    530.    530.    530.    530.    530.    530.    530.

SPECIES      WJ      BR      GS      PY      OS      LO      CY      BL      EO      WO
SDI MAX     530.    530.    530.    530.    530.    530.    530.    530.    530.    530.

SPECIES      BO      VO      IO      BM      BU      RA      MA      GC      DG      FL
SDI MAX     530.    530.    530.    530.    530.    530.    530.    530.    530.    530.

SPECIES      WN      TO      SY      AS      CW      WI      CN      CL      OH
SDI MAX     530.    530.    530.    530.    530.    530.    530.    530.    530.

                                  NATIONAL VOLUME ESTIMATOR LIBRARY EQUATION NUMBERS
SPECIES CUBIC FOOT BOARD FOOT SPECIES CUBIC FOOT BOARD FOOT SPECIES CUBIC FOOT BOARD FOOT SPECIES CUBIC FOOT BOARD FOOT 
------- ---------- ---------- ------- ---------- ---------- ------- ---------- ---------- ------- ---------- ---------- 
  PC    632BEHW000 632BEHW000   IC    632BEHW081 632BEHW081   RC    632BEHW242 632BEHW242   WF    632BEHW015 632BEHW015
  RF    632BEHW020 632BEHW020   SH    632BEHW021 632BEHW021   DF    632BEHW202 632BEHW202   WH    632BEHW263 632BEHW263
  MH    632BEHW264 632BEHW264   WB    632BEHW101 632BEHW101   KP    632BEHW103 632BEHW103   LP    632BEHW108 632BEHW108
  CP    632BEHW000 632BEHW000   LM    632BEHW113 632BEHW113   JP    632BEHW116 632BEHW116   SP    632BEHW117 632BEHW117
  WP    632BEHW119 632BEHW119   PP    632BEHW122 632BEHW122   MP    632BEHW000 632BEHW000   GP    632BEHW000 632BEHW000
  WJ    632BEHW064 632BEHW064   BR    632BEHW000 632BEHW000   GS    632BEHW000 632BEHW000   PY    632BEHW231 632BEHW231
  OS    632BEHW298 632BEHW298   LO    632BEHW000 632BEHW000   CY    632BEHW000 632BEHW000   BL    632BEHW000 632BEHW000
  EO    632BEHW000 632BEHW000   WO    632BEHW815 632BEHW815   BO    632BEHW818 632BEHW818   VO    632BEHW000 632BEHW000
  IO    632BEHW000 632BEHW000   BM    632BEHW312 632BEHW312   BU    632BEHW000 632BEHW000   RA    632BEHW351 632BEHW351
  MA    632BEHW361 632BEHW361   GC    632BEHW431 632BEHW431   DG    632BEHW492 632BEHW492   FL    632BEHW000 632BEHW000
  WN    632BEHW000 632BEHW000   TO    632BEHW631 632BEHW631   SY    632BEHW000 632BEHW000   AS    632BEHW746 632BEHW746
  CW    632BEHW747 632BEHW747   WI    632BEHW920 632BEHW920   CN    632BEHW000 632BEHW000   CL    632BEHW000 632BEHW000
  OH    632BEHW998 632BEHW998

SITECODE   SITE INDEX INFORMATION:
            PC=  116.;  IC=   90.;  RC=  103.;  WF=  129.;  RF=  129.;  SH=  129.;  DF=  129.;  WH=  123.; MH=  116.
            WB=  116.;  KP=  116.;  LP=  116.;  CP=  116.;  LM=  116.;  JP=  121.;  SP=  129.;  WP=  121.; PP=  121.
            MP=  116.;  GP=  116.;  WJ=   98.;  BR=   98.;  GS=  129.;  PY=   52.;  OS=   98.;  LO=   36.; CY=   54.
            BL=   44.;  EO=   36.;  WO=   52.;  BO=   72.;  VO=   98.;  IO=   36.;  BM=   98.;  BU=   72.; RA=   98.
            MA=   98.;  GC=   98.;  DG=   52.;  FL=   90.;  WN=   52.;  TO=   98.;  SY=   98.;  AS=   52.; CW=   98.
            WI=   32.;  CN=   32.;  CL=   32.;  OH=   72.
           SITE SPECIES=DF CODE=    7

INVENTORY POINT CROSS REFERENCE (FVS SEQUENTIAL POINT NUMBER = POINT NUMBER AS ENTERED IN THE INPUT DATA):
  1=       1;   2=       0
----------------------------------------------------------------------------------------------------------------------------------


                                                     ACTIVITY SCHEDULE

STAND ID= SWOTEST                       MGMT ID= NONE     Stand SWOTEST at test or                                               

----------------------------------------------------------------------------------------------------------------------------------

CYCLE  DATE  EXTENSION  KEYWORD   DATE  PARAMETERS:
-----  ----  ---------  --------  ----  ------------------------------------------------------------------------------------------

   1   2010
               BASE     TREELIST  2010        3.00      0.00
               CMPU     BA        2010        0.00
               CMPU     TPA       2010        0.00
               CMPU     QMD       2010        0.00
               CMPU     TOPHT     2010        0.00
               CMPU     TCUFT     2010        0.00
               CMPU     BDFT      2010        0.00

   2   2015
               BASE     TREELIST  2015        3.00      0.00
               CMPU     BA        2015        0.00
               CMPU     TPA       2015        0.00
               CMPU     QMD       2015        0.00
               CMPU     TOPHT     2015        0.00
               CMPU     TCUFT     2015        0.00
               CMPU     BDFT      2015        0.00

   3   2020
               BASE     TREELIST  2020        3.00      0.00
               CMPU     BA        2020        0.00
               CMPU     TPA       2020        0.00
               CMPU     QMD       2020        0.00
               CMPU     TOPHT     2020        0.00
               CMPU     TCUFT     2020        0.00
               CMPU     BDFT      2020        0.00

   4   2025
               BASE     TREELIST  2025        3.00      0.00
               CMPU     BA        2025        0.00
               CMPU     TPA       2025        0.00
               CMPU     QMD       2025        0.00
               CMPU     TOPHT     2025        0.00
               CMPU     TCUFT     2025        0.00
               CMPU     BDFT      2025        0.00

   5   2030
               BASE     TREELIST  2030        3.00      0.00
               CMPU     BA        2030        0.00
               CMPU     TPA       2030        0.00
               CMPU     QMD       2030        0.00
               CMPU     TOPHT     2030        0.00
               CMPU     TCUFT     2030        0.00
               CMPU     BDFT      2030        0.00

   6   2035
               BASE     TREELIST  2035        3.00      0.00
               CMPU     BA        2035        0.00
               CMPU     TPA       2035        0.00
               CMPU     QMD       2035        0.00
               CMPU     TOPHT     2035        0.00
               CMPU     TCUFT     2035        0.00
               CMPU     BDFT      2035        0.00

   7   2040
               BASE     TREELIST  2040        3.00      0.00
               CMPU     BA        2040        0.00
               CMPU     TPA       2040        0.00
               CMPU     QMD       2040        0.00
               CMPU     TOPHT     2040        0.00
               CMPU     TCUFT     2040        0.00
               CMPU     BDFT      2040        0.00

   8   2045
               BASE     TREELIST  2045        3.00      0.00
               CMPU     BA        2045        0.00
               CMPU     TPA       2045        0.00
               CMPU     QMD       2045        0.00
               CMPU     TOPHT     2045        0.00
               CMPU     TCUFT     2045        0.00
               CMPU     BDFT      2045        0.00

   9   2050
               BASE     TREELIST  2050        3.00      0.00
               CMPU     BA        2050        0.00
               CMPU     TPA       2050        0.00
               CMPU     QMD       2050        0.00
               CMPU     TOPHT     2050        0.00
               CMPU     TCUFT     2050        0.00
               CMPU     BDFT      2050        0.00

  10   2055
               BASE     TREELIST  2055        3.00      0.00
               CMPU     BA        2055        0.00
               CMPU     TPA       2055        0.00
               CMPU     QMD       2055        0.00
               CMPU     TOPHT     2055        0.00
               CMPU     TCUFT     2055        0.00
               CMPU     BDFT      2055        0.00
----------------------------------------------------------------------------------------------------------------------------------
CALLED ORGANON.DLL::PREPARE::BETA, CYCLE= 1 IERROR= 0


CALIBRATION STATISTICS:


                                                 WF    IC    SP    PP    PY    RC    WH    BM    RA    MA    GC

                                                 DG    TO    CY    WO    BO    WI    DF
                                                ----  ----  ----  ----  ----  ----  ----  ----  ----  ----  ----

NUMBER OF RECORDS PER SPECIES                      2     1     1     1     1     1     1     1     1     1     1

                                                   1     1     1     1     1     1     7

NUMBER OF RECORDS WITH MISSING HEIGHTS             0     0     0     0     0     0     0     0     0     0     0

                                                   0     0     0     0     0     0     0

NUMBER OF RECORDS WITH BROKEN OR DEAD TOPS         0     0     0     0     0     0     0     0     0     0     0

                                                   0     0     0     0     0     0     0

NUMBER OF RECORDS WITH MISSING CROWN RATIOS        0     0     0     0     0     0     0     0     0     0     0

                                                   0     0     0     0     0     0     0

NUMBER OF RECORDS AVAILABLE FOR SCALING
THE DIAMETER INCREMENT MODEL                       0     0     0     0     0     0     0     0     0     0     0

                                                   0     0     0     0     0     0     0

RATIO OF STANDARD ERRORS
(INPUT DBH GROWTH DATA : MODEL)                 1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00

                                                1.00  1.00  1.00  1.00  1.00  1.00  1.00

WEIGHT GIVEN TO THE INPUT GROWTH DATA WHEN
DBH GROWTH MODEL SCALE FACTORS WERE COMPUTED    0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00

                                                0.00  0.00  0.00  0.00  0.00  0.00  0.00

INITIAL SCALE FACTORS FOR THE
DBH INCREMENT MODEL                             1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00

                                                1.00  1.00  1.00  1.00  1.00  1.00  1.00

NUMBER OF RECORDS AVAILABLE FOR SCALING
THE SMALL TREE HEIGHT INCREMENT MODEL             0     0     0     0     0     0     0     0     0     0     0

                                                  0     0     0     0     0     0     0

INITIAL SCALE FACTORS FOR THE SMALL TREE
HEIGHT INCREMENT MODEL                          1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00  1.00

                                                1.00  1.00  1.00  1.00  1.00  1.00  1.00

NUMBER OF RECORDS WITH MISTLETOE                   0     0     0     0     0     0     0     0     0     0     0

                                                   0     0     0     0     0     0     0        FOREST VEGETATION SIMULATOR     VERSION      -- ORGANON BETA RELEASE                   RV:11/14/12    05-09-2013  15:32:27

STAND ID: SWOTEST                       MGMT ID: NONE     Stand SWOTEST at test or


STAND POLICIES:   All, SOUTH_RIVER_2, 10002

                                        STAND COMPOSITION (BASED ON STOCKABLE AREA)
--------------------------------------------------------------------------------------------------------------------------
                            PERCENTILE POINTS IN THE
                     DISTRIBUTION OF STAND ATTRIBUTES BY DBH      TOTAL/ACRE
        STAND       -----------------------------------------      OF STAND         DISTRIBUTION OF STAND ATTRIBUTES BY
YEAR  ATTRIBUTES      10     30     50     70     90    100       ATTRIBUTES       SPECIES AND 3 USER-DEFINED SUBCLASSES
----  -----------   ------ ------ ------ ------ ------ ------   --------------  ------------------------------------------
                                 (DBH IN INCHES)



2010  TREES           5.0    5.0   15.0   25.0   35.0    35.0      44. TREES     57.% DF1,  23.% DF2,   2.% WF1,   1.% IC1
      VOLUME:  
        TOTAL        20.0   25.0   30.0   35.0   35.0    35.0    4250. CUFT      98.% DF1,   2.% DF2,   0.% RA1,   0.% WF1
        MERCH        20.0   25.0   30.0   35.0   35.0    35.0    4250. CUFT      98.% DF1,   2.% DF2,   0.% RA1,   0.% WF1
        MERCH        20.0   30.0   30.0   35.0   35.0    35.0   21155. BDFT      99.% DF1,   1.% DF2,   0.% ---,   0.% ---


      ACCRETION      15.0   25.0   30.0   35.0   35.0    35.0      78. CUFT/YR   97.% DF1,   3.% DF2,   0.% RA1,   0.% WF1
      MORTALITY       5.0   15.0   20.0   30.0   35.0    35.0       1. CUFT/YR   78.% DF1,  13.% DF2,   2.% WI1,   1.% TO1



2015  TREES           5.6    6.3   16.5   26.5   36.4    36.4      43. TREES     58.% DF1,  23.% DF2,   2.% WF1,   1.% DG1
      VOLUME:  
        TOTAL        21.5   26.5   31.5   36.4   36.4    36.4    4632. CUFT      98.% DF1,   2.% DF2,   0.% RA1,   0.% WF1
        MERCH        21.5   26.5   31.5   36.4   36.4    36.4    4632. CUFT      98.% DF1,   2.% DF2,   0.% RA1,   0.% WF1
        MERCH        21.5   26.5   31.5   36.4   36.4    36.4   23366. BDFT      99.% DF1,   1.% DF2,   0.% ---,   0.% ---


      ACCRETION      16.5   26.5   31.5   31.5   36.4    36.4     171. CUFT/YR   93.% DF1,   6.% DF2,   0.% WF1,   0.% RA1
      MORTALITY       6.3   16.5   21.5   31.5   36.4    36.4       1. CUFT/YR   79.% DF1,  11.% DF2,   2.% WI1,   1.% TO1



2020  TREES           6.1    7.5   18.0   27.9   37.8    37.8      43. TREES     58.% DF1,  23.% DF2,   2.% WF1,   1.% BM1
      VOLUME:  
        TOTAL        23.0   27.9   32.9   37.8   37.8    37.8    5481. CUFT      97.% DF1,   2.% DF2,   0.% WF1,   0.% RA1
        MERCH        23.0   27.9   32.9   37.8   37.8    37.8    5481. CUFT      97.% DF1,   2.% DF2,   0.% WF1,   0.% RA1
        MERCH        23.0   27.9   32.9   37.8   37.8    37.8   28874. BDFT      99.% DF1,   1.% DF2,   0.% ---,   0.% ---


      ACCRETION      18.0   23.0   27.9   32.9   37.8    37.8     182. CUFT/YR   92.% DF1,   7.% DF2,   0.% WF1,   0.% RA1
      MORTALITY       7.2   18.0   23.0   32.9   37.8    37.8       1. CUFT/YR   73.% DF1,  14.% DF2,   2.% SP1,   2.% RA1



2025  TREES           7.0   14.3   19.4   29.3   39.2    39.2      42. TREES     59.% DF1,  23.% DF2,   2.% WF1,   1.% BM1
      VOLUME:  
        TOTAL        24.4   29.3   34.2   39.2   39.2    39.2    6383. CUFT      97.% DF1,   3.% DF2,   0.% WF1,   0.% RA1
        MERCH        24.4   29.3   34.2   39.2   39.2    39.2    6383. CUFT      97.% DF1,   3.% DF2,   0.% WF1,   0.% RA1
        MERCH        24.4   29.3   34.2   39.2   39.2    39.2   33913. BDFT      98.% DF1,   2.% DF2,   0.% WF1,   0.% SP1


      ACCRETION      19.4   24.4   29.3   34.2   39.2    39.2     192. CUFT/YR   91.% DF1,   8.% DF2,   0.% WF1,   0.% PP1
      MORTALITY       8.2   14.3   24.4   29.3   39.2    39.2       2. CUFT/YR   69.% DF1,  16.% DF2,   3.% SP1,   2.% RA1



2030  TREES           7.6   15.7   20.8   30.6   40.4    40.4      42. TREES     60.% DF1,  23.% DF2,   2.% WF1,   1.% BM1
      VOLUME:  
        TOTAL        20.8   30.6   35.5   40.4   40.4    40.4    7336. CUFT      96.% DF1,   4.% DF2,   0.% WF1,   0.% RA1
        MERCH        20.8   30.6   35.5   40.4   40.4    40.4    7336. CUFT      96.% DF1,   4.% DF2,   0.% WF1,   0.% RA1
        MERCH        25.7   30.6   35.5   40.4   40.4    40.4   40494. BDFT      97.% DF1,   2.% DF2,   0.% WF1,   0.% SP1